public/index.html

```diff
@@ -4,13 +4,15 @@
 		<meta charset="UTF-8" />
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<!-- <script src="https://cdn.tailwindcss.com"></script> -->
```

Comment:
this comment too

Reply:
I removed this  coz right now I am using configure tailwind not cdn